deploy/rbac.yaml

```diff
@@ -36,7 +36,7 @@ rules:
     - list
 # allow reading Deployments and StatefulSets to get matching Pods for VPAs
 - apiGroups: ["apps"]
-  resources: ["deployments", "statefulsets"]
+  resources: ["deployments", "statefulsets", "replicasets", "daemonsets"]
```

Comment:
Thanks for the suggestion, the actual implementation is missing for ReplicaSet/DaemonSet. I'm adding this now in #159,